### ISSUE TITLE
PanelBar: k-content element has "region" role with no label associated

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#5955]
### Bug report

Currently, the `k-content` elements in the PanelBar have `region` role. As per the WAI-ARIA specification (https://www.w3.org/WAI/PF/aria/roles#region):

> Authors SHOULD ensure that a region has a heading referenced by aria-labelledby. This heading is provided by an instance of the standard host language heading element or an instance of an element with role heading that contains the heading text.
> When defining regions of a web page, authors are advised to consider using standard document landmark roles. If the definitions of these regions are inadequate, authors can use the region role and provide the appropriate accessible name.

Related to: #5954 

### Reproduction of the problem
Review the generated HTML on the PanelBar Index demo:
https://demos.telerik.com/kendo-ui/panelbar/index

### Expected/desired behavior
We should reconsider the use of the `region` role or reference a heading as per the requirements.

### Environment
* **Kendo UI version:** 2020.2.617
* **Browser:** [all]

**Comment by veselints** [comment]
Research on the issue and the suggested approach are available here:
https://github.com/telerik/kendo-ui-core/issues/5954#issuecomment-692697993

**Comment by veselints** [comment]
`menu` and `menuitem` roles have been changed to `tree` and `treeitem` respectively. The `region` role has been removed. Here are the screen readers changes after that change:

### NVDA in Firefox
* The role of the widget is announced as a `treeview`;
* The level of the currently focused item is announced;
* The nested items within an expanded item are not announced when their parent gets focus;
